templates/displayApplicants.phtml

```diff
@@ -38,7 +38,7 @@
                             </div>
 
                         <th class="col-xs-2">Cohort date
```

Comment:
can't see the closing tag for this but it's probs under the </div> so i'm sure it's fine.